service/build.gradle

```diff
@@ -21,21 +21,58 @@ repositories {
 }
 
 dependencies {
-    implementation "org.springframework.boot:spring-boot-starter-web:2.3.1.RELEASE"
```

Comment:
Did you downgrade starter-web version intentionally? If so, can we keep all the other SpringBoot dependencies in the same version?